manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/metadata.yaml

```diff
@@ -4,33 +4,33 @@ metadata:
   labels:
     app: metadata
     app.kubernetes.io/name: {{ .Release.Name }}
-  name: metadata-service
+  name: metadata-grpc-service
```

Comment:
Any reason on why prefer renaming it?

Is it more for understand the meaning? I agree with you that it should be named like this PR. Background: rename deployment in manifests is treated as not-back-compatibile in GCP Marketplace. It means we need to bump version. If just to make the name be more understandable, I may prefer you hold this CL or ~1-2 weeks so that we put it to KFP 0.4.x

Reply:
The purpose is to be consistent with kubeflow deployment.

We've released 0.3.0, even if we hold this, next release won't be backward compatible. So I think there is no reason to hold it.